### ISSUE TITLE
[Bug] Song not playing in linux build

### ISSUE THREAD

**Issue by Reimu24** [Drewol/unnamed-sdvx-clone#669]
**Describe the bug**
everything works fine except when started a song,
nothing happens

**To Reproduce**
Start any song

**Expected behavior**
Song starts playing

**Screenshots**
![Screenshot_20240510_154924_TermuxX11](https://github.com/Drewol/unnamed-sdvx-clone/assets/124783767/e0e749bd-d4b6-45bf-8b45-fb796858b9c1)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.4
 - GPU: Running udroid on S22+, Adreno 730
 - CPU: 8 gen 1
 - Version: Git clone 2024.5.10

**Additional context**

**Log file**
If applicable, attach the log file or paste a snippet of it.
[log_usc-game.txt](https://github.com/Drewol/unnamed-sdvx-clone/files/15271977/log_usc-game.txt)

**Comment by Drewol** [comment]
Does audio play at all in any of the menus?

**Comment by Reimu24** [comment]
nope, also by song not playing i mean the entire chart not playing too

**Comment by Drewol** [comment]
The game syncs the chart playback with the audio but if there is no audio then it cant play. So the issue is with the game not being able to open any audio device.

**Comment by coyote963** [comment]
Had this same issue and reinstalled SDL2 fixed it
